pkgs/development/libraries/mapbox-gl-native/default.nix

```diff
@@ -35,6 +35,7 @@ mkDerivation rec {
       url = "https://aur.archlinux.org/cgit/aur.git/plain/fix-compilation.patch?h=mapbox-gl-native";
       hash = "sha256-KgJHyoIdKdnQo+gedns3C+mEXlaTH/UtyQsaYR1T3iI=";
     })
+    ./0001-Fix-narrowing-conversion-warning-error.patch
```

Comment:
We prefer using `fetchpatch` over commiting unmodified patches into nixpkgs.

https://github.com/mapbox/mapbox-gl-native/commit/2955d0e479f57a39a0af4a0fa7ca7683455cca58.patch

Reply:
Wasn't aware that was an option for an unmerged PR. Done. Thanks!